@xen-orchestra/web-core/lib/components/title/VtsTitle.vue

```diff
@@ -0,0 +1,39 @@
+<!-- v2 -->
+<template>
+  <div class="vts-title">
+    <div class="typo h4-medium label">
+      <slot />
+    </div>
+    <div class="actions">
+      <slot name="actions" />
+    </div>
```

Comment:
Since it is an optional slot, it is better to check the "presence" of the slot to avoid having an empty `<div>` in the HTML when the slot is not used. Two solutions:
1. With `$slots`:
    ```html
    <div v-if="$slots.actions" class="actions">
      <slot name="actions" />
    </div>
    ```
2. With `slots` variable defined in the script section:
    ```html
    <template>
        <div v-if="slots.actions" class="actions">
          <slot name="actions" />
        </div>
    </template>

    <script setup lang="ts">
    const slots = defineSlots<{ ... }>()
    </script>
    ```
    See [this related discussion](https://github.com/vatesfr/xen-orchestra/pull/7993#discussion_r1800558578)